### PR TITLE
Decidim courses: admin section

### PR DESCRIPTION
**Pull request by rbngzlv**

### CODE REVIEW

File: decidim-courses/app/forms/decidim/courses/admin/course_form.rb [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+        attribute :hero_image
+        attribute :remove_banner_image
+        attribute :remove_hero_image
+
+        validates :scope, presence: true, if: proc { |object| object.scope_id.present? }
+        validates :slug, presence: true, format: { with: Decidim::Course.slug_format }
+
+        validate :slug_uniqueness
+
+        validates :modality, inclusion: { in: ::Decidim::Course::MODALITIES }
```
Comment (victorol1) [inline decidim-courses/app/forms/decidim/courses/admin/course_form.rb:41]:
```suggestion
        validates :modality, presence: true, inclusion: { in: ::Decidim::Course::MODALITIES }
```

**Review by victorol1 — CHANGES_REQUESTED** [review]
LGTM, just a small change and it's ready, also be careful with ruby version, it seems that your workspace is ignoring the .ruby-version file.

File: decidim-courses/app/forms/decidim/courses/admin/course_form.rb (hunk 1 above)
Comment (rbngzlv) [reply decidim-courses/app/forms/decidim/courses/admin/course_form.rb]: I'm developing with docker, and the image used in the project ( `decidim/decidim:0.22.0`) have the version `2.6` inside it. I'll revert the change to the `Gemfile.lock` and try to not push this change again.

**Comment by rbngzlv** [comment]
I'm developing with docker, and the image used in the project ( decidim/decidim:0.22.0) have the version 2.6 inside it. I'll revert the change to the Gemfile.lock and try to not push this change again.

**Review by tramuntanal — APPROVED** [review]

**Comment by tramuntanal** [comment]
@rbngzlv and @victorol1  , Decidim 0.24 will come with Ruby 2.7.1. We'll try to use the higher Ruby version as possible in this project (compatible with Decidim).

**Pull request merged**